build.gradle

```diff
@@ -18,9 +18,7 @@ apply plugin: 'io.spring.dependency-management'
 apply plugin: 'jacoco'
 
 jar {
-    baseName = 'lancie-api'
-    version = '2017.1.6'
```

Comment:
Should we really remove this?

Reply:
The jar file is only used in the container, and the container originates from a specific build with a specific tag. I think we no longer need it. This was needed when there were multiple jar files in one archive directory on the deployment server.

Reply:
We could change to this format? But I'm not quite sure when it's needed.

https://github.com/WISVCH/events/blob/develop/build.gradle#L24-L26